products/sle/main.pm

```diff
@@ -481,7 +481,6 @@ sub load_cluster_boot {
     # Standard boot and configuration
     loadtest "qa_automation/patch_and_reboot" if is_updates_tests;
     loadtest "console/consoletest_setup"      if is_updates_tests;
-    loadtest "console/hostname";
```

Comment:
Is possible to don't remove it?

Reply:
yes :)

Reply:
huh? Either you want to fool us or you fooled yourself :D the whole `load_cluster_boot` is scheduled exactly once when `HPC` is set. Now you don't schedule "console/hostname` when HPC is set. Does not make sense

Reply:
no it's actually work as expected. @czerw plan to use this function in another places where it will need hostname, but in case of HPC we doing same things which happening in hostname in different place

Reply:
When @czerw wants to use it then he should add the test module. As of now the code does not make sense and we should avoid to [add functionality early](http://www.extremeprogramming.org/rules/early.html)

Reply:
@asmorodskyi @okurz I also think, if is boot functionality moved to boot_hdd_image, we can drop function load_cluster_boot completely

So maybe https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/4631 should be merged first?